unquote.go

```diff
@@ -85,12 +85,9 @@ raw:
 			} else if c == escapeChar {
 				buf.WriteString(input[0 : len(input)-len(cur)-l])
 				if os.PathSeparator == escapeChar {
-					next := rune(cur[0])
-					switch next {
-					case singleChar, doubleChar, escapeChar, 'n':
```

Comment:
Just for the record: so we sacrifice the possibility to use `\` to escape single or double quotes on Windows. For example: open a new unnamed file, press Ctrl-S to save it, and in the `Filename:` prompt, type `\"`. Result: without this PR the file is successfully saved as `"`, while with this PR we get `Error parsing arguments: Unterminated double-quoted string`.

I think it's ok, and it's actually better if `\` behaves consistently, i.e. never escapes on Windows. And the user can still escape quotes in other ways: `"'"` and `'"'`.